src/convertToObject.js

```diff
@@ -1,18 +1,24 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .filter(value => value.includes(':'))
+    .map(value => value.split(':'))
+    .reduce((key, value) => {
+      key[value[0].trim()] = value[1].trim();
```

Comment:
Let's use destructuring like these
```suggestion
    .reduce((styles, [key, value]) => {
      styles[key.trim()] = value.trim();
```